src/test/java/com/commercetools/sync/commons/utils/ResourceCustomUpdateActionUtilsTest.java

```diff
@@ -96,8 +98,13 @@ void buildResourceCustomUpdateActions_WithNullOldCustomFields_ShouldBuildUpdateA
         final Category oldCategory = mock(Category.class);
         when(oldCategory.getCustom()).thenReturn(null);
 
-        final CategoryDraft newCategoryDraft = CategorySyncMockUtils.getMockCategoryDraft(Locale.ENGLISH, "name",
-            "key", "parentId", "customTypeId", new HashMap<>());
+        final CategoryDraft newCategoryDraft =
+            CategoryDraftBuilder.of(LocalizedString.ofEnglish("name"), LocalizedString.ofEnglish("testSlug"))
```

Comment:
Why not using the method `CategorySyncMockUtils.getMockCategoryDraft`?

Reply:
It's used for creating a mock with type id (to created directly without sync methods), so I use the typical draft builder, creating a common utility is not needed for this case.